server/controller/employee/addNewEmployee.js

```diff
@@ -0,0 +1,15 @@
+const postNewEmployee = require('../../database/query/employee/postNewEmployee');
+
+const addNewEmployee = (req, res) => {
+	const Empdata = req.body;

```

Comment:
`employeeData`

- PascalCase is reserverd for classes in javascript (https://www.robinwieruch.de/javascript-naming-conventions)